packages/web-components/src/spinner/spinner.styles.ts

```diff
@@ -12,81 +17,170 @@ import {
 } from '../styles/states/index.js';
 
 export const styles = css`
-  ${display('flex')}
+  ${display('inline-flex')}
 
   :host {
-    display: flex;
+    --size: 32px;
+    --duration: 1.5s;
+    --indicatorSize: calc(0.09375 * var(--size));
```

Comment:
Does this scale better than pixel values for the different sizes?

Reply:
Not necessarily. I was trying to map to the previous scaling behavior of a 1.5px stroke in a 16x16 viewBox (0.09375 ratio). We could "round up" to `.1` and hardcode the sizes if that's easier.

Reply:
Ok. I went ahead and hardcoded this to the values that the React implementation uses. It's a lot more clear and less mysterious about what width we're shooting for. Uses `strokeWidth` tokens to explicitly set the border/stroke width.